crates/uv/tests/pip_install.rs

```diff
@@ -756,24 +726,15 @@ fn install_editable() -> Result<()> {
     Installed 4 packages in [TIME]
      + anyio==4.3.0
      + idna==3.6
-     + poetry-editable==0.1.0 (from file://[WORKSPACE_DIR]/scripts/packages/poetry_editable)
+     + poetry-editable==0.1.0 (from file://[WORKSPACE]/scripts/packages/poetry_editable)
      + sniffio==1.3.1
     "###
     );
 
     // Install it again (no-op).
-    uv_snapshot!(filters, Command::new(get_bin())
-        .arg("pip")
-        .arg("install")
+    uv_snapshot!(context.filters(), command(&context)
         .arg("-e")
-        .arg("../../scripts/packages/poetry_editable")
-        .arg("--strict")
-        .arg("--cache-dir")
-        .arg(context.cache_dir.path())
-        .arg("--exclude-newer")
-        .arg(EXCLUDE_NEWER)
-        .env("VIRTUAL_ENV", context.venv.as_os_str())
-        .env("CARGO_TARGET_DIR", "../../../target/target_install_editable"), @r###"
+        .arg(context.workspace_root.join("scripts/packages/poetry_editable")), @r###"
```

Comment:
There are various instances where the `command` utility was not being used — I'm not sure why but where I see it I've addressed it. Another pass will probably be needed to fix remaining cases.